controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -405,6 +405,15 @@ func (r *OCSInitializationReconciler) reconcileUXBackendService(initialData *ocs
 						IntVal: 8888,
 					},
 				},
+				{
```

Comment:
why do we need to expose upstream here, then it'll be like any other service which any client can access and ask for an onboarding token?

let me know otherwise.

Reply:
AFAIK we need this as we are using the service name in the upstream URL